_posts/2025-04-29-release-notes-3.3.6.md

```diff
@@ -0,0 +1,31 @@
+---
+category: release
+permalink: /news/3.3.6/
+title: "Scala 3.3.6 LTS is now available!"
+---
+Scala 3.3.6 LTS is now available!
+
+This patch release backports most of the improvements and bugfixes introduced in the Scala Next series up to the Scala 3.6.4 release.
+All of the backported changes were proven to not break either binary or source compatibility, by testing over 1500 projects in the [Scala 3 Open Community Build](https://github.com/VirtusLab/community-build3).
+
+# Breaking changes
+
+- Aligned `-Wconf` parsing order with Scala 2 semantics: `-Wconf:x,y` now means `-Wconf:x -Wconf:y`, with y overruling x, rather than the reverse to align with Scala 3.4+ and with user intuition [#21985](https://github.com/scala/scala3/pull/21985)
```

Comment:
```suggestion
- Aligned `-Wconf` parsing order with Scala 2.13.15+ semantics: `-Wconf:x,y` now means `-Wconf:x -Wconf:y`, with y overruling x, rather than the reverse to align with Scala 3.4+ and with user intuition [#21985](https://github.com/scala/scala3/pull/21985)
```

Reply:
Ach, actually, somehow I copy pasted it. This was already in 3.3.5